docs/use cases/README.md

```diff
@@ -238,7 +366,52 @@ TODO: PM_03
 ```md
 @startuml
 
-TODO: PM_04
+ left header
+
+        <font color=000 size=18><b>ID:</b> PM_04
+
+        <font color=000 size=16><b>НАЗВА:</b> Переглянути прогрес проекту
+        <font color=000 size=16><b>УЧАСНИКИ:</b> Менеджер проектів, Система
+
+        <font color=000 size=16><b>ПЕРЕДУМОВИ:</b>
+        <font color=000 size=16>Наявність проекту
+
+        <font color=000 size=16><b>РЕЗУЛЬТАТ:</b> Відображення прогресу виконання проекту у відсотках
+
+        <font color=000 size=16><b>ВИКЛЮЧНІ СИТУАЦІЇ:</b>
+        <font color=000 size=16>Відсутні
+        
+        <font color=000 size=16><b>ОСНОВНИЙ СЦЕНАРІЙ:</b>
+
+    end header
+
+    |Користувач|
+        start
+        :Користувач відкриває 
+        вікно з усіма проектами;
```

Comment:
Would be better kinda:
1. User clicks the button "Projects"
2. The system is sending projects list to user

...